server/testutils/factories/assessmentSearchParameters.ts

```diff
@@ -3,5 +3,5 @@ import { AssessmentSearchParameters } from '@approved-premises/ui'
 import { faker } from '@faker-js/faker/locale/en_GB'
 
 export default Factory.define<AssessmentSearchParameters>(() => ({
-  crn: `C${faker.number.int({ min: 100000, max: 999999 })}`,
+  query: faker.lorem.lines(),
```

Comment:
This would generate several lines of text, which seems counterintuitive here -- wouldn't this best generate either a CRN or a first name, for instance?
```suggestion
  query: faker.helpers.arrayElement([
    `C${faker.number.int({ min: 100000, max: 999999 })}`,
    faker.person.firstName(),
  ]),
```